src/backend/mailbox_manager/tests/test_admin_actions.py

```diff
@@ -204,3 +204,83 @@ def test_fetch_domain_expected_config__should_not_fetch_for_disabled_domain(clie
     assert "Domains disabled are excluded from fetch" in response.content.decode(
         "utf-8"
     )
+
+
+@responses.activate
+@pytest.mark.django_db
+def test_send_pending_mailboxes(client):
```

Comment:
You may add an already enabled domain to check the message about already activated

Reply:
I'm not sure which "already activated" message are you referring to. 

As this tests simulate a situation where mailboxes that fell out of sync on an enabled domain, I already use an enabled domain (+ pending mailboxes).

Did you want me to add a "nothing to sync" message when the domain is enabled + 0 pending mailboxes ?

Reply:
Ignore my comment, I made a confusion between mail and domain I guess ^^